addons/pause/userscript.js

```diff
@@ -72,15 +72,11 @@ export default async function ({ addon, global, console, msg }) {
       pausedThreadState = new WeakMap();
     }
   };
-
-  const originalStepToProcedure = vm.runtime.sequencer.stepToProcedure;
-  vm.runtime.sequencer.stepToProcedure = function (thread, proccode) {
-    if (proccode.startsWith("sa-pause")) {
-      setPaused(true);
-      return;
-    }
-    return originalStepToProcedure.call(this, thread, proccode);
-  };
+  function pauseBlock(role, target) {
+    setPaused(true);
+  }
+  addon.tab.addBlock("sa-pause", [], pauseBlock, true);
+  addon.tab.addBlock("breakpoint", [], pauseBlock);
```

Comment:
what's the difference between these two blocks

Reply:
Nothing, one's for backwards compatability

Reply:
> Nothing, one's for backwards compatability

Does the name need to be changed? If it does, the `sa-` prefix should be re-added, otherwise it would break projects with a block called "breakpoint".